frontend/app/templates/project/show/transportation/tdf/trip-results.hbs

```diff
@@ -1,12 +1,10 @@
 <div class="row">
   <div class="sixteen wide column sticky-transportation-bar">
-    {{#sticky-element top=79}}
-      <Transportation::LandUseMenu
-        @showTotalLink={{true}}
-        @tdfSection="trip-results"
-        @model={{model}}
-      />
-    {{/sticky-element}}
+    <Transportation::LandUseMenu
+      @showTotalLink={{true}}
+      @tdfSection="trip-results"
+      @model={{model}}
+    />
```

Comment:
Why move this into the component? It makes us less able to compose things together. Element stickiness is now a mandatory part of <Transportation::LandUseMenu/>